src/js/player.js

```diff
@@ -880,6 +880,29 @@ class Player extends Component {
     this.on(this.tech_, 'posterchange', this.handleTechPosterChange_);
     this.on(this.tech_, 'textdata', this.handleTechTextData_);
 
+    /**
+     * Fires when the video's intrinsic width and/or height has changed.
+     * This differs from Player#resize, which is triggered when the player
+     * is resized.
+     *
+     * @event player#videoresize
```

Comment:
I think `player#videoresize` will have to be `Player#videoresize` with a capital P.